### DISCUSSION TITLE
Grouped List always renders expanded.

### DISCUSSION THREAD

**Discussion by billgewrgoulas** [microsoft/fluentui#28521] [General] [answered]
Hi, is there any way to collapse all the groups in the GroupedList at start, because now after the first render the GroupedList opens all the groups.

**Comment by spmonahan** [comment, marked as answer]
Yes, you can make all the groups collapsed by default by settings `isCollapsed` to `true` for each `IGroup`.

Here's a Codepen example: https://codepen.io/seanms/pen/YzdqZMe?editors=0010

There's a lot going on in the Implementation section of `GroupedList` so it's easy to miss but for more reference check out the implementation for `IGroup`: https://developer.microsoft.com/en-us/fluentui#/controls/web/groupedlist#implementation

**Reply by billgewrgoulas** [reply]
Thanks
